src/layers/spaces.js

```diff
@@ -0,0 +1,99 @@
+import FeatureLayer from '@arcgis/core/layers/FeatureLayer';
+import accessibleIcon from '../icons/accessible-icon.svg';
+import chargingStationIcon from '../icons/charging-station.svg';
+import circleDollarToSlotIcon from '../icons/circle-dollar-to-slot.svg';
+import circleXmarkIcon from '../icons/circle-xmark.svg';
+import massachusettsIcon from '../icons/massachusetts.svg';
+import motorcycleIcon from '../icons/motorcycle.svg';
+import peopleCarryBoxIcon from '../icons/people-carry-box.svg';
+import peopleGroupIcon from '../icons/people-group.svg';
+import personWalkingLuggageIcon from '../icons/person-walking-luggage.svg';
+import squareParkingIcon from '../icons/square-parking.svg';
+import truckRampBoxIcon from '../icons/truck-ramp-box.svg';
```

Comment:
So these I just kinda ripped from fontawesome and put here (probably) as placeholders since I do think the previous icons were a little deficient (most of them were just... colored letters).

Later on in the roadmap (if we get that far) I'd be open to fleshing them out a bit more, but I don't care to do that as I'm just pushing up a bunch of my "exploratory" work for now.